multigeiger/version.h

```diff
@@ -1,9 +1,8 @@
-// Revision history:
+const char* software_version = "1.11";
```

Comment:
Is it only x.y because we have no space on the small display?

A x.y.z scheme would be better so we can follow semantic versioning (== use .z for maintenance bug fixes made after initial x.y.0 release).

Reply:
I would rather remove the date from display and have .z displayed.

Reply:
The reason for not to have z is the extra byte for lora - if we have z, we must send 3 bytes. And in lora it is important so send as few bytes as possible.
And I think, the date should be on the display. We should discuss this with Jürgen @WwwBoehriDe and Nicola @ecocurious.

Reply:
not necessarily.

1.2.3 could be encoded as 0x1023 - assuming that we will neved need major version > 15 or patch version > 15.

Not many patch versions can be ensured by releasing often.

Major version change means "big change" or even "breaking change" in semantic versioning - maybe we never ever need that.

Reply:
(just an example with nice hex representation - we could use also octal 0oXYYZZ (== 15 bits))